src/validations/rules/ssp.sch

```diff
@@ -1616,6 +1616,62 @@
                 role="error"
                 test="current() = $security-objective-levels">[Section B Check 3.10] A FedRAMP SSP must specify an allowed security objective
                 value.</sch:assert>
+            <sch:let
+                name="impactName"
+                value="substring-after(substring-after(local-name(), '-'), '-')" />
+            <sch:let
+                name="impactSelected"
+                value="string-join(/o:system-security-plan/o:system-characteristics/o:system-information//o:information-type//*[contains(local-name(), $impactName)]/o:selected/text())" />
+            <sch:let
+                name="impactBase"
+                value="string-join(/o:system-security-plan/o:system-characteristics/o:system-information//o:information-type//*[contains(local-name(), $impactName)]/o:base/text())" />
+            <sch:let
+                name="securityImpactLevelSelected"
+                value="
+                    if (matches($impactSelected, 'high'))
+                    then
+                        ('fips-199-high')
+                    else
+                        (if (matches($impactSelected, 'moderate'))
+                        then
+                            ('fips-199-moderate')
+                        else
+                            (if (matches($impactSelected, 'low'))
+                            then
+                                ('fips-199-low')
+                            else
+                                ('')))" />
+            <sch:let
+                name="securityImpactLevelBase"
+                value="
+                    if (matches($impactBase, 'high'))
+                    then
+                        ('fips-199-high')
+                    else
+                        (if (matches($impactBase, 'moderate'))
+                        then
+                            ('fips-199-moderate')
+                        else
+                            (if (matches($impactBase, 'low'))
+                            then
+                                ('fips-199-low')
+                            else
+                                ('')))" />
+            <sch:let
+                name="impactValue"
+                value="
+                    if ($securityImpactLevelSelected ne '')
+                    then
+                        ($securityImpactLevelSelected)
+                    else
+                        ($securityImpactLevelBase)" />
+            <sch:assert
```

Comment:
This assertion fails when an information-type CIA impact set is (na).

Reply:
In one of the supporting documents there is reference to any value other than the fips-199-* should be output as fips-199-low.  I cannot put my finger on it at the moment.

However, I noticed that the *-impact and security-objective-* elements both already have a test for each containing an appropriate value as defined by the fedramp_values.xml document.  So I thought that defining the else condition as 'fips-199-low' was excessive.

However, for potential clarity during maintenance I added the else condition value.